deployer/deployer-services/src/main/java/io/hyscale/deployer/services/model/DeployerActivity.java

```diff
@@ -32,6 +32,8 @@
 	POD_INITIALIZED("Pod initialization "),
 	POD_CREATION("Pod creation "),
 	POD_READINESS("Pod readiness "),
+	POD_SCHEDULED("Pod scheduled"),
+	PODSCHEDULED("PodScheduled"),
```

Comment:
Do we need Pod scheduled? As Pod initialized was taking care of that only

Reply:
No need of explicit state for Pod Scheduling , we deal with creation, initialization, readiness.